### PR TITLE
release note

### PR DESCRIPTION
**Pull request by chicm-ms**

### CODE REVIEW

File: docs/en_US/Release.md [hunk 1]
```diff
@@ -1,5 +1,109 @@
...
+* Engineering Improvements
+  - For remote training service,  trial jobs require no GPU are now scheduled with round-robin policy instead of random.
+* Web Portal & User Experience
+  - Support user to add customized trial.
+  - User can zoom out/in in detail graphs, except Hyper-parameter.
+
+### Bug fix
+  - Fix the table sort issue when failed trials haven't metrics. -Issue #1773
+  - Maintain selected status(Maximal/Minimal) when the page switched. -PR#1710
+  - Make hyper-parameters graph's default metric yAxis more accurate. -PR#1736
```
Comment (QuanluZhang) [inline docs/en_US/Release.md]: could add GPU related bug fix: permission, scheduling in remote mode

File: docs/en_US/Release.md [hunk 2]
```diff
@@ -1,5 +1,109 @@
...
+### Major Features
+* Feature Engineering
+  - New feature engineering interface
+  - Feature selection algorithms: Gradient feature selector & GBDT selector
+* NAS
+  - New NAS interface
+  - NAS algorithms: ENAS, DARTS, P-DARTS (in PyTorch)
+  - NAS in classic mode (each trial runs independently)
+* Model compression
+  - New model pruning algorithms: lottery ticket pruning approach, L1Filter pruner, Slim pruner, FPGM pruner
```
Comment (QuanluZhang) [inline docs/en_US/Release.md]: better to add reference link for the algorithms

File: docs/en_US/Release.md [hunk 3]
```diff
@@ -1,5 +1,109 @@
...
+
+### Major Features
+* Feature Engineering
+  - New feature engineering interface
+  - Feature selection algorithms: Gradient feature selector & GBDT selector
+* NAS
+  - New NAS interface
+  - NAS algorithms: ENAS, DARTS, P-DARTS (in PyTorch)
+  - NAS in classic mode (each trial runs independently)
+* Model compression
```
Comment (QuanluZhang) [inline docs/en_US/Release.md]: support the API for exporting compressed model

File: docs/en_US/Release.md [hunk 4]
```diff
@@ -1,5 +1,109 @@
...
+  - New model pruning algorithms: lottery ticket pruning approach, L1Filter pruner, Slim pruner, FPGM pruner
+  - New model quantization algorithms: QAT quantizer, DoReFa quantizer
+* Training Service
+Support OpenPAI token authentication
+* Examples:
+  - A new MNIST trial example supports tensorflow 2.0.
```
Comment (QuanluZhang) [inline docs/en_US/Release.md]: could add the rocksdb example

File: docs/en_US/Release.md [hunk 5]
```diff
@@ -1,5 +1,109 @@
...
+* Engineering Improvements
+  - For remote training service,  trial jobs require no GPU are now scheduled with round-robin policy instead of random.
```
Comment (QuanluZhang) [inline docs/en_US/Release.md]: could add docstring improvement, pylint improvement

File: docs/en_US/Release.md (hunk 1 above)
Comment (chicm-ms) [reply docs/en_US/Release.md]: Thandks, added.

File: docs/en_US/Release.md (hunk 3 above)
Comment (chicm-ms) [reply docs/en_US/Release.md]: Thanks, added.

File: docs/en_US/Release.md (hunk 5 above)
Comment (chicm-ms) [reply docs/en_US/Release.md]: Thanks, added.

File: docs/en_US/Release.md (hunk 4 above)
Comment (chicm-ms) [reply docs/en_US/Release.md]: Thanks, added.

File: docs/en_US/Release.md (hunk 2 above)
Comment (chicm-ms) [reply docs/en_US/Release.md]: Thanks, added.

File: docs/en_US/Release.md [hunk 6]
```diff
@@ -1,5 +1,116 @@
...
+  - [EfficientNet PyTorch example](https://github.com/ultmaster/EfficientNet-PyTorch)
+  - [Cifar10 NAS example](https://github.com/microsoft/nni/blob/v1.1/examples/trials/nas_cifar10/README.md)
+* [Model compression toolkit - Alpha release](https://github.com/microsoft/nni/blob/v1.1/docs/en_US/Compressor/Overview.md): We are glad to announce the alpha release for model compression toolkit on top of NNI, it's still in the experiment phase which might evolve based on usage feedback. We'd like to invite you to use, feedback and even contribute
+
+### Fixed Bugs
+
+* Multiphase job hangs when search space exhuasted (issue #1204)
+* `nnictl` fails when log not available (issue #1548)
+
+## Release 1.0 - 2/9/2019
```
Comment (ultmaster) [inline docs/en_US/Release.md]: 2019/2/9 -> 2019/9/2?

**Pull request closed without merging**